starter-code/src/functions-and-arrays.js

```diff
@@ -1,23 +1,68 @@
 // Find the maximum
+function maxOfTwoNumbers(numberOne, numberTwo) {
+  if (numberOne > numberTwo){
+    return numberOne;
+  }
+    else if (numberTwo > numberOne){
+      return numberTwo;
+    }
+    else {
+      return numberOne;
+    }
+  }
+
 
 // Finding Longest Word
-var words = [
-  'mystery',
-  'brother',
-  'aviator',
-  'crocodile',
-  'pearl',
-  'orchard',
-  'crackpot'
-];
+
+function findLongestWord() {
+  
+  var words = [
+    'mystery',
+    'brother',
+    'aviator',
+    'crocodile',
+    'pearl',
+    'orchard',
+    'crackpot'
+  ];
+  
+  var longest= "";
+  
+  for (i=0; i<words.length; i++){
+      if (words[i].length > longest.length){
+        longest= words[i];
+      }
+  }
+  
+  words.forEach(function(words){
+  });
+    
+  return longest;
+  
+  }
 
 // Calculating a Sum
 
 var numbers = [6, 12, 1, 18, 13, 16, 2, 1, 8, 10];
 
+var sum = numbers.reduce(sumArray, 0);
+
+function sumArray(a, b) {
+  return a + b;
+}
```

Comment:
Can you make sum function without using .reduce()? Can you think of solution using for or forEach loop? :)